<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvailabilityWindow</class>
 <widget class="QMainWindow" name="AvailabilityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messdienerplan - Abmeldungen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_start_day">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abmeldung Von</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_single_day">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Einzelabmeldung</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Grades">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Churchservers">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_end_day">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abmeldung Bis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_save">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_remove_day">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Abmeldungen Entfernen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_start_day">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Start Abmeldung</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
